apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -173,55 +171,127 @@ export class SendMessage {
     });
 
     switch (stepType) {
-      case StepTypeEnum.SMS:
-        return await this.sendMessageSms.execute(sendMessageCommand);
-      case StepTypeEnum.IN_APP:
-        return await this.sendMessageInApp.execute(sendMessageCommand);
-      case StepTypeEnum.EMAIL:
-        return await this.sendMessageEmail.execute(sendMessageCommand);
-      case StepTypeEnum.CHAT:
-        return await this.sendMessageChat.execute(sendMessageCommand);
-      case StepTypeEnum.PUSH:
-        return await this.sendMessagePush.execute(sendMessageCommand);
-      case StepTypeEnum.DIGEST:
-        return await this.digest.execute(command);
-      case StepTypeEnum.DELAY:
-        return await this.sendMessageDelay.execute(command);
-      case StepTypeEnum.CUSTOM:
-        return await this.executeStepCustom.execute(sendMessageCommand);
+      case StepTypeEnum.SMS: {
+        await this.sendMessageSms.execute(sendMessageCommand);
+        break;
+      }
+      case StepTypeEnum.IN_APP: {
+        await this.sendMessageInApp.execute(sendMessageCommand);
+        break;
+      }
+      case StepTypeEnum.EMAIL: {
+        await this.sendMessageEmail.execute(sendMessageCommand);
+
+        break;
+      }
+      case StepTypeEnum.CHAT: {
+        await this.sendMessageChat.execute(sendMessageCommand);
+
+        break;
+      }
```

Comment:
Line-break consistency.
```suggestion
      case StepTypeEnum.EMAIL: {
        await this.sendMessageEmail.execute(sendMessageCommand);
        break;
      }
      case StepTypeEnum.CHAT: {
        await this.sendMessageChat.execute(sendMessageCommand);
        break;
      }
```